### ISSUE TITLE
Can’t get To-single relationship field ID in labelResolver

### ISSUE THREAD

**Issue by zamkevich** [keystonejs/keystone#3057]
## Describe
Wen I use a To-single relationship Field **with Many** option
and then tried to get it ID's in labelResolver it comes only one time.

All other fields ID or values came but the Field ID comes only one time.
Without option Many all ok.

## Bug
console.log(item) right in labelResolver

![image](https://user-images.githubusercontent.com/13717428/83085273-d1b18280-a0ce-11ea-8cd5-5b4268cf432a.png)

## Fields conf
![image](https://user-images.githubusercontent.com/13717428/83085274-d5dda000-a0ce-11ea-85bd-a2f43f39fdc6.png)
![image](https://user-images.githubusercontent.com/13717428/83085285-daa25400-a0ce-11ea-8a8d-1098b71f2965.png)

**Comment by Vultraz** [comment]
Is `Clinic_position` a valid ref? Not `Clinic.position`?

**Comment by zamkevich** [comment]
Yes, look at the first image, this field is correctly displayed when creating the first record.

If many option is false - all fine.

**Comment by Vultraz** [comment]
And which list has the `labelResolver`?

**Comment by zamkevich** [comment]
Full list module (simplified code) 

```
module.exports = function (){
  return { 
    fields: {
     // That trouble field 
      positions: { 
        type: Relationship,
        many: true,
        ref: 'Clinic_position', 
      },
      clinic: { 
        type: Relationship,
        ref: 'Clinic.staff', 
      },
      doctor: {
        type: Relationship,
        ref: 'Doctor.employer', 
      },
    },
    labelResolver: async (item) => {
        console.log(item)
    }
}

```

**Comment by MichaelZaporozhets** [comment]
I'm experiencing the same issue- indeed, Relationship fields `many: true` will not be passed into the labelResolver function.

**Comment by zamkevich** [comment]
@MichaelZaporozhets 

For now, you can achieve it by the custom handler function

**Comment by MichaelZaporozhets** [comment]
@zamkevich Cheers- yeah, I worked around the issue; but it's still a bug that should be looked at :)
